ocp_resources/node_network_configuration_policy.py

```diff
@@ -345,6 +345,12 @@ def status(self):
                 return condition["reason"]
 
     def wait_for_configuration_conditions_unknown_or_progressing(self, wait_timeout=30):
+        if (
+            self.status
+            and self.status == self.Conditions.Reason.SUCCESSFULLY_CONFIGURED
```

Comment:
if you change nncp, this may exit before configuration changes started and may end up not waiting for the config to start